### ISSUE TITLE
Medic class some bugs.

### ISSUE THREAD

**Issue by AlexanderCurl** [doctorluk/Reign-of-the-Undead-Revolution-Scripts#7]
As I seen, when I trying the indev mod with the actually up-to-date scripts got some bugs.
1. The Mini UZI shoots too fast like somebody using rapid-fire or something like that.
2. The G3 is not shooting automatically when you press and hold the button, but it should auto weapon.

**Comment by doctorluk** [comment]
Yeah, that is most likely caused by the ongoing weapons overhaul. Until then the weapon configuration will be bad.
